packages/web3/src/api/api-explorer.ts

```diff
@@ -136,6 +136,15 @@ export interface ExplorerInfo {
   lastFinalizedInputTime: number
 }
 
+export interface FungibleTokenMetadata {
+  /** @format 32-byte-hash */
+  id: string
+  symbol: string
+  name: string
+  /** @format uint256 */
+  decimals: string
```

Comment:
😢

Reply:
yeah I know, but as we discussed it's safer for the backend to return a BigNumer as that's what we get from the node.

Reply:
I know, I know, I'm still sad tho :P

Reply:
I'll get over it, don't worry 😆